test/extended/cpu_partitioning/crio.go

```diff
@@ -37,7 +37,7 @@ container_ids=$(sudo crictl ps -q)
 container_data=$(sudo crictl inspect $container_ids)
 
 # Filter down CRIO data to relevant information only
-workload_containers=$(echo $container_data | jq -rs '[ 
+workload_containers=$(echo $container_data | jq '[
```

Comment:
I am so sorry for this, but can you make it backward compatible? Fallback to one command if the other reports an error? 

Rationale is, if we do have to revert rhcos because something else shows up, then this test is now dependent on the new code. Backward compat, we're safe either way. Can it be done?

Reply:
No need to be sorry, you're absolutely right, updated to check.